consul/src/main/java/com/linecorp/armeria/common/consul/ConsulConfigSetters.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.consul;
+
+import com.linecorp.armeria.internal.consul.ConsulClientBuilder;
+
+/**
+ * Sets properties for building {@code ConsulClient}.
```

Comment:
We should not mention a class in an internal package because we hide them all.

```suggestion
 * Sets properties for building a Consul client.
```

Reply:
Done.